.ldrelease/zip-helper.ps1

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
This is just a copy of the logic we needed from the built-in Releaser helper scripts. As long as this implementation is being done on a per-project basis rather than being part of our project templates, I figured it was best not to rely on details of those built-ins.